### ISSUE TITLE
beam search, temperature, video segments

### ISSUE THREAD

**Issue by ISHITADG** [OpenGVLab/Ask-Anything#43]
Can someone please help me understand what effect does parameters like beam search, temperature and video segments have on the VQA responses ? Could not find this in the repo or InternVideo/Vchat paper.

**Comment by Andy1621** [comment]
For `beam search` and `temperature`, they are the hyperparameters for LLM. Here are some descriptions from Bing：
> In LLM, beam search is a decoding strategy that generates a set of beam sequences to choose from. Diverse beam search is an extension of the beam search strategy that allows for generating a more diverse set of beam sequences to choose from¹. Temperature is a hyperparameter used in the softmax function that controls the randomness of the output. A higher temperature will generate more random output¹².

¹: [Hugging Face](https://huggingface.co/docs/transformers/generation_strategies)
²: [GitHub](https://github.com/openai/whisper/discussions/549)

For the `video segment`, we use it to control the input frames. Large video segment means inputting more frames.
